core/src/main/java/com/backblaze/b2/json/B2Json.java

```diff
@@ -245,10 +282,15 @@ public String listToJson(List<?> list, Class<?> valueClass) throws B2JsonExcepti
      * if there is anything but whitespace after the JSON value.
      */
     public <T> T fromJsonUntilEof(InputStream in, Class<T> clazz) throws IOException, B2JsonException {
-        return fromJsonUntilEof(in, clazz, 0);
+        return fromJsonUntilEof(in, clazz, B2JsonOptions.DEFAULT);
+    }
+
+    @Deprecated
```

Comment:
willing to add a comment to @Deprecateds which say what to use instead? even if it's generic, like "use the B2JsonOptions version instead"?

Reply:
ok